From: Hector Garcia <hector@debian.org>

fix hyphen-used-as-minus-sign problem:

This manual page seems to contain a hyphen where a minus sign was
intended. By default, "-" chars are interpreted as hyphens (U+2010) by
groff, not as minus signs (U+002D). Since options to programs use
minus signs (U+002D), this means for example in UTF-8 locales that you
cannot cut and paste options, nor search for them easily. The Debian
groff package currently forces "-" to be interpreted as a minus sign
due to the number of manual pages with this problem, but this is a
Debian-specific modification and hopefully eventually can be removed.

"-" must be escaped ("\-") to be interpreted as minus. If you really
intend a hyphen (normally you don't), write it as "\(hy" to emphasise
that fact. See groff(7) and especially groff_char(7) for details, and
also the thread starting with
http://lists.debian.org/debian-devel/2003/debian-devel-200303/msg01481
.html

If you use some tool that converts your documentation to groff format,
this tag may indicate a bug in the tool. Some tools convert dashes of
any kind to hyphens. The safe way of converting dashes is to convert
them to "\-".

Because this error can occur very often, Lintian shows only the first
10 occurrences for each man page and give the number of suppressed
occurrences. If you want to see all warnings, run Lintian with the
-d/--debug option.

Refer to the groff_char(7) manual page for details

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>
---
 lircd.8 |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/doc/man/lircd.8
+++ b/doc/man/lircd.8
@@ -120,7 +120,7 @@ was given a button in the lircd config f
 named KEY_1, the '1' key code will be generated. You will find a
 complete list of possible button names in /usr/include/linux/input.h.
 
-The --repeat-max option sets an upper limit to the number of repeats
+The \-\-repeat-max option sets an upper limit to the number of repeats
 when sending a signal. The current default is 600. A SEND_START
 request will repeat the signal this many times. Also, if the number of
 repeats in a SEND_ONCE request exceeds this number, it will be
